### DISCUSSION TITLE
Sort Topics by Overall Age

### DISCUSSION THREAD

**Discussion by weslambert** [community/community#3235] [Discussions]
It would be great if could sort by overall topic age (to help with prioritizing response), as opposed to the "last updated" topic sorting available through the `New` option.

**Comment by evi-liu** [comment]
Thanks for the feedback! We are tracking this request our end and gauging customer impact via upvotes on this post.

**Reply by raygesualdo** [reply]
Not having this is becoming a large pain point for us. We're using Discussions as a documentation portal as well, but without being able to sort from newest to oldest (by creation date), a lot of context is lost as to what the most recently written articles are.

**Reply by markseuffert** [reply]
I would be interested too. [For example here are news](https://github.com/datenstrom/yellow/discussions/categories/see-what-s-new), where sorting by creation date would be helpful. It would be lovely to look through the announcements of our community in a chronological order. I think it would make it more intuitive to find past announcements, e.g. product releases, as I remember them in the order they were posted.

![screenshot-2022-01](https://user-images.githubusercontent.com/2429165/151703437-48f2f766-0fc9-47c9-b73c-336f267ca128.png)

**Reply by markseuffert** [reply]
It looks like the filter `sort:date_created` is available now. [Here's an example category sorted by date](https://github.com/datenstrom/yellow/discussions/categories/see-what-s-new?discussions_q=category%3A%22See+what%27s+new%22+sort%3Adate_created). Nice work.

**Comment by PranamBhat** [comment]
Hello!

We have :

"New"
"Top : Today"
"Top : Past week"
"Top : Past month"
"Top : Past year"
"Top : All"
"Answered"
"Unanswered"

All these sorting is based on voting-counts. Especially when it comes to active/inactive topics, with the help of vote-counts and "New" filter we can easily sort out old and new discussion topics.

I think this is sufficient.

Thank you!

**Reply by ann0see** [reply]
I wouldn't say this is sufficiant for e.g. technical support. If a support question of somebody is missed because there are too many other discussions going on, this user will not get help. Adding a filter like the proposed solution might give him an answer.

**Reply by weslambert** [reply]
Exactly -- it can sometimes be hard to keep track otherwise, and can easily lead to folks getting missed from (not just an answer, but) a response perspective.

**Reply by ann0see** [reply]
'Totally agree. We're moving from SourceForge forums to GitHub Discussions, but this is one of the most annoying missing functions. 

See: https://sourceforge.net/p/llcon/discussion/533517/thread/096167a246/?page=3&limit=25#2270

I hope more users would vote for this! Actually I think this should be quite easy to implement.

**Reply by TheMode** [reply]
This is definitely not sufficient, just confusing. There should be a way to show discussions in a consistent order.

**Comment by tomcrane** [comment]
Even if you don't wish to clutter or confuse the UI with additional sort options, you could still support the same set that _issues_ supports in queries (and hence in the URL as a query string parameter), e.g., `sort:created-asc` - useful for giving someone a link "read these discussions..."
